docs/pre-release/reference/config.md

```diff
@@ -20,6 +20,9 @@ timeouts:
   intercept: 10s
 logLevels:
   userDaemon: debug
+images:
+  registry: privateRepo
+  agentImage: ambassador-telepresence-agent:1.8.0
```

Comment:
we probably should also call out how to configure the registry in the helm charts

Reply:
So I have this line in `helm.md`: `For more details on what the Helm chart installs and what can be configured, take a look at the Helm chart [README](https://github.com/telepresenceio/telepresence/tree/release/v2/charts/telepresence).`
 that redirects to the chart readme that has all the values that can be configured.  
 
 I did that so that we don't have to remember to edit the README and then the docs, do you think that's sufficient?

Reply:
hmm maybe then just a note in the `Add license to cluster` section in the `cluster-config.md` document, stating something like "if you use `helm` to install the traffic manager, see the chart's [README](https://github.com/telepresenceio/telepresence/tree/release/v2/charts/telepresence) to learn how to configure the image registry and license secret"